README.md

```diff
@@ -1,2 +1,10 @@
-# Adafruit.lplib
-Adafruit boards. Inofficial.
+# Adafruit
+
+## Description
+
+This is a [LibrePCB](https://librepcb.org) library!
+Just edit this file to add a description about it.
```

Comment:
If you want, add a description here :wink:

Reply:
Done.